graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -15,6 +17,11 @@
 ABSOLUTE_SELECTIVITY = 'absolute'
 FRACTIONAL_SELECTIVITY = 'fractional'
 
+INEQUALITY_OPERATORS = frozenset(['<', '<=', '>', '>=', 'between'])
+
+MIN_UUID_INT = 0
+MAX_UUID_INT = 2**128-1
```

Comment:
where do these numbers come from?

Reply:
UUIDs are 128-bit numbers: https://en.wikipedia.org/wiki/Universally_unique_identifier

Reply:
https://en.wikipedia.org/wiki/Universally_unique_identifier
I couldn't find a `MAX_UUID` constant, so I had to make do with harcoding it.
Once we decide to implement histograms (mentioned in a TODO), we can just use histograms instead to inform ourselves about min and max values, and forgo these harcoded variables.